katello.dot

```diff
@@ -27,27 +27,27 @@ node "server" {
     [Foreman Discovery] as foreman_discovery
     [Foreman SCAP] as foreman_scap
 
-    foreman_tasks -down-> dynflow
-    katello -> foreman
-    katello -> foreman_tasks
-    katello -> foreman_docker
+    foreman_tasks -down..> dynflow
+    katello ..> foreman
+    katello ..> foreman_tasks
+    katello ..> foreman_docker
```

Comment:
Ah, didn't realize we were trying to imply a requirement.  I guess its okay as is.